treeherder/perf/models.py

```diff
@@ -95,6 +100,10 @@ class Meta:
             # particular set of properties
             ('repository', 'suite', 'test', 'framework', 'application',
```

Comment:
I just realized that the new `application` property I've added in bug 1593190 to this old unique constraint is basically making it useless. We need to remove it from the definition of all `PerformanceSignature`'s unique constraints.

Reply:
@sarah-clements could you find the time to merge & deploy this today? Especially because it contains the hotfix for removing the `application` property from [this line](https://github.com/mozilla/treeherder/blob/master/treeherder/perf/models.py#L96).

Reply:
I don't *think* duplicated data could have slid in, while this [old unique constraint](https://github.com/mozilla/treeherder/blob/master/treeherder/perf/models.py#L96-L97) was made useless.

If the hotfix from either PRs fails on deploy, then it's a sign that duplicated data did slide in; good thing is it's easy to recover from a failed deploy like this & then it'll be the 1st thing I'll tackle in the morning.